adal/src/main/java/com/microsoft/aad/adal/AuthenticationDialog.java

```diff
@@ -195,7 +195,7 @@ public void run() {
 
         public DialogWebViewClient(Context ctx, String stopRedirect,
                 AuthenticationRequest request) {
-            super(ctx, stopRedirect, request);
+            super(ctx, stopRedirect, request, null);
```

Comment:
i will file an issue and do this later

---
In reply to: [92700697](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r92700697) [](ancestors = 92700697)